float64 rpm
float64 current
float64 temperature